kernel/src/process/credentials/credentials_.rs

```diff
@@ -75,28 +77,28 @@ impl Credentials_ {
     //  ******* Uid methods *******
 
     pub(super) fn ruid(&self) -> Uid {
-        self.ruid.get()
+        self.ruid.load(Ordering::Acquire)
     }
 
     pub(super) fn euid(&self) -> Uid {
-        self.euid.get()
+        self.euid.load(Ordering::Acquire)
     }
 
     pub(super) fn suid(&self) -> Uid {
-        self.suid.get()
+        self.suid.load(Ordering::Acquire)
     }
 
     pub(super) fn fsuid(&self) -> Uid {
-        self.fsuid.get()
+        self.fsuid.load(Ordering::Acquire)
     }
 
     pub(super) fn set_uid(&self, uid: Uid) {
         if self.is_privileged() {
-            self.ruid.set(uid);
-            self.euid.set(uid);
-            self.suid.set(uid);
+            self.ruid.store(uid, Ordering::Release);
+            self.euid.store(uid, Ordering::Release);
+            self.suid.store(uid, Ordering::Release);
         } else {
-            self.euid.set(uid);
+            self.euid.store(uid, Ordering::Release);
```

Comment:
Use `Relaxed` for `ruid`, `euid`, etc. `Acquire` and `Release` are used to establish happens-before relationships between concurrent code. But no user code of these fields use them for synchronization. So `Relaxed` is enough. This arguments apply to other atomic integer-like types.

Reply:
OK. I'll use `Relaxed`. Actually I kept the previous choice of `Ordering` because I didn't see into the related code and I was not sure if the choice is of some specific use.